Client/Public/JS/Controllers/GameController.js

```diff
@@ -701,6 +745,7 @@ function joinGameSession(game_password) {
     window.setTimeout(function() {
         socket.emit('update_current_room_name');
         initModelUpgrades(modelUpgrades);
+        updateDisplayedGameProgress();
```

Comment:
NTS-3: Test to see if we can just call 'initModelGameProgress' and get rid of some code